[14.0][Add] microsoft outlook single-tenant support (#998)
OCA/social: microsoft_outlook_single_tenant/readme/DESCRIPTION.rst

> Currently odoo only supports multi-tenant apps authetication in azure.

Reviewer: ```suggestion
Currently odoo only supports multi-tenant apps authentication in azure.
```

> Currently odoo only supports multi-tenant apps authentication in azure.
> That module alows you configure odoo as a single-tenant app.

Reviewer 2: ```suggestion
By default Odoo only supports multi-tenant apps authentication on Azure.
This module allows to configure Odoo as a single-tenant app.
```

Author: Updated the `DESCRIPTION`
